sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/input_output.py

```diff
@@ -57,6 +57,7 @@ class PrimitiveOutputSchema(OutputPortSchema):
         required=True,
     )
     is_control = fields.Bool()
+    early_available = fields.Bool()
```

Comment:
may add a private preview env protection here

Reply:
hide it under `is_private_preview_enabled`
there is one thing different from previous init/finalize: output schema will throw error when private preview env is not set, and as schema classes are always imported at the very beginning (before test class), we need to reload related classes (including schema and entity) in fixture. I did it for `early_available` in new added fixture `enable_early_available_output`.